python/instance.py

```diff
@@ -1,10 +1,14 @@
 from __future__ import annotations
 
 import dataclasses
-from typing import List, Iterable, Iterator
+from typing import Iterable, Iterator, List, TYPE_CHECKING
 
-from point import Point
 import parse
+from point import Point
+from svg import SVGGraphic
+
+if TYPE_CHECKING:
```

Comment:
This is a meme:
```python
# Constant that's True when type checking, but False here.
TYPE_CHECKING = False
```
To avoid a circular import.

Reply:
Not a huge fan of circular dependencies, but I think it's fine for now. I would prefer a trait-based system where Instance and Solution impl SvgSerialization or something, but that is too much work. (Same with regular text serialization.)